specreduce/core.py

```diff
@@ -2,3 +2,246 @@
 # This module implements the base CCDPROC functions
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
```

Comment:
general comment for this whole file: i would suggest putting flux calibration specific utilities in their own file, e.g. `fluxcal.py` or `flux_calibration.py`. `mag2flux()` may be appropriate to stay and be made public.

Reply:
FWIW that's how I've been structuring my version of this exact same update

Reply:
Yeah, @jradavenport. This has been a draft based on your `pydis` source code, except I cleaned it up a bit.

Reply:
For reference, here's where my version currently is
https://github.com/jradavenport/specreduce/blob/workflow_exploration/specreduce/fluxcal.py
We're definitely doing duplicate work right now, LOL

Reply:
Yeah @jradavenport or you could open a new PR and take it from here. I could close this once your version is merged.

Reply:
Or we could collaborate on this PR while keeping it open.